.github/workflows/check_code.yml

```diff
@@ -56,7 +56,7 @@ jobs:
           # https://stackoverflow.com/a/16081218
           EXIT_STATUS=0
           poetry run pytest --cov=pyartifactory --cov-branch --cov-report=xml || EXIT_STATUS=$?
-          poetry run python-codacy-coverage -r coverage.xml || EXIT_STATUS=$?
```

Comment:
If you use their automatic bash script you can remove the dependency here https://github.com/anancarv/python-artifactory/blob/master/pyproject.toml#L61